.golangci.yaml

```diff
@@ -14,6 +14,7 @@ linters-settings:
       - github.com/gogo/protobuf/proto
       - github.com/google/certificate-transparency-go/trillian
       - github.com/google/certificate-transparency-go/x509
+      - github.com/google/certificate-transparency-go/asn1
       - encoding/asn1
       - crypto/x509
```

Comment:
Should these stdlib packages be on the block list?

Reply:
I wonder why they weren't actually being blocked? You've recently added crypto/x509 imports without complaint from the linter right?

Reply:
The linter only runs against *new* code.

Reply:
Those imports you added were new though, weren't they?